frontend/src/index.jade

```diff
@@ -31,8 +31,10 @@ html
           )
           .timestamp-footer
             span Generated by &nbsp;
-            a(href="https://github.com/reposense/RepoSense")
-              strong RepoSense
+            a(
```

Comment:
Does this follow the style guide?

Reply:
The eslint in CI does not give warning, so I assume it is correct.